client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -223,10 +227,19 @@ export class CommentDetailAuthor extends Component {
 		return (
 			<div className={ classes }>
 				<div className="comment-detail__author-preview">
-					<Gravatar user={ {
-						avatar_URL: this.props.authorAvatarUrl,
-						display_name: this.props.authorDisplayName,
-					} } />
+					<div className="comment-detail__author-avatar">
+						<div className="comment-detail__author-avatar">
+							{ 'comment' === commentType &&
```

Comment:
Nitpick (optional): maybe we could move this to `getCommentImage` (or some more appropriate name) helper function for readability. We are also doing something similar bellow in `comment-detail-header.jsx` where it could also be reused.